projects/ngx-interactive-paycard-demo/src/assets/credit-card.ts

```diff
@@ -0,0 +1,71 @@
+export const creditCards: any = [
```

Comment:
Please don't use `any` typed objects. Please create the according `class`/`interface` for it.

Reply:
no problem, I remove this part until fix the problems.